### DISCUSSION TITLE
Dashboard 전역상태관리 (v)

### DISCUSSION THREAD

**Discussion by un0211** [Part3-Team15/taskify#135] [General]
@wjsdncl 이제 더이상 전역 관리했던 dashboard 쓰는 곳 없나요??
만약 없다면 리팩토링 하면서 dashboard 전역 상태 관련 코드 다 지워버려도 될 것 같아요~~

**Comment by wjsdncl** [comment]
저는 사용 안하고 있습니다 🤔

**Reply by wjsdncl** [reply]
전에 사용했다가 페이지네이션 하느라 뺐었어요

**Comment by wayandway** [comment]
맞아요 이제 안쓰니 지워놓으면 좋겠네요!!

**Comment by un0211** [comment]
제가 이슈 만들어서 다 없애버릴게요! ㅋㅋㅋ
제가 초기에 폴더 만드느라 만든 getDashboards(빈파일)도 자꾸 거슬려서...
